classes/04class/exercises/c04-iac04/gepifanio/terraform-code/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "region" {
+    type    = string
+    default = "ap-southeast-2"
+}
+
+variable "key_pair_name" {
+    type    = string
+}
+
+variable "ami" {
+    type    = string
+}
+variable "subnets" {
+    type    = string
+}
+variable "asg_policy_up" {
+    type    = string
+}
+variable "asg_policy_down" {
+    type    = string
+}
```

Comment:
These variables are the capacity change, right?

Something is missing here. Maybe variables with the thresholds to trigger the policies, e.g. CPU utilization. 

Have a look on how ASG scaling policies are triggered:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_metric_alarm#example-in-conjunction-with-scaling-policies